Added release notes and documentation updates for Deploy 4.9.3 10.3.3, 12.1.3 13.0.3. (#5762)
umbraco/UmbracoDocs: 10/umbraco-deploy/getting-started/deploy-settings.md

When this is in place a translation for a language that doesn't exist in the target environment will be ignored. A warning message will be output to the log.

### SetEmptyDictionaryItemsOnTransfer

> When deploying dictionary items, Umbraco Deploy will follow the approach used for all content, which is that empty values are transferred and set.
> 
> As such, if you transfer a dictionary item with an empty translation to another environment that already contains a translation, it will be overwritten.

Reviewer: ```suggestion
When deploying dictionary items, Umbraco Deploy follows the approach used for all content, emptying values that are transferred and set.

If you transfer a dictionary item with an empty translation to another environment that already contains a translation, it will be overwritten.
```
